subsys/net/lib/wifi_mgmt_ext/wifi_mgmt_ext.c

```diff
@@ -73,7 +73,8 @@ static int __stored_creds_to_params(struct wifi_credentials_personal *creds,
 
 	/* If channel is set to 0 we default to ANY. 0 is not a valid Wi-Fi channel. */
 	params->channel = (creds->header.channel != 0) ? creds->header.channel : WIFI_CHANNEL_ANY;
-	params->timeout = CONFIG_WIFI_MGMT_EXT_CONNECTION_TIMEOUT;
+	params->timeout = (creds->header.timeout != 0) ? creds->header.timeout :
```

Comment:
what will be the default value? If you use `0` it will be timeout immediately, it should be `-1`

Reply:
Default is set to CONFIG_WIFI_MGMT_EXT_CONNECTION_TIMEOUT (30). If there is no timeout set in creds database, connection timeout is set to 30.